provisioner/aws-manager.js

```diff
@@ -507,12 +865,11 @@ AwsManager.prototype._trackNewSpotRequest = function (sr) {
   // AWS Api when we submit the SpotRequest
   assert(sr);
 
-  var that = this;
-
   var allKnownSrIds = this.knownSpotInstanceRequestIds();
 
   if (!allKnownSrIds.includes(sr.request.SpotInstanceRequestId)) {
-    var filtered = objFilter(sr.request, that.filters.spotReq);
+    // XXX var filtered = objFilter(sr.request, that.filters.spotReq);
+    var filtered = sr.request;
```

Comment:
Yes.  I needed a couple new properties, and those are properites that might not be present in yet-unfulfilled spot requests and living instances, which objFilter doesn't currently handle cleanly.  I'll be turning this back on if memory usage increases dramatically.